Support/CMake/Toolchain-Windows-x86_64-Toolset-x86_64.cmake

```diff
@@ -0,0 +1,11 @@
+##
```

Comment:
It seems to me this file is unused. You should get rid of it.

Generally speaking, in this project, we don't really care about which toolset is being used, so not specifying it at all and letting cmake pick the default seems like a better solution.

Reply:
Dead.